cmd/podman/pods/rm.go

```diff
@@ -94,6 +94,9 @@ func removePods(namesOrIDs []string, rmOptions entities.PodRmOptions, printIDs b
 	responses, err := registry.ContainerEngine().PodRm(context.Background(), namesOrIDs, rmOptions)
 	if err != nil {
 		setExitCode(err)
+		if rmOptions.Force && registry.GetExitCode() == 1 {
+			registry.SetExitCode(define.ExecErrorCodeIgnore)
```

Comment:
This should only happen if the error is pod does not exist, and this is the only error returned.